### PR TITLE
Adding Contribution Guidelines

### PR DESCRIPTION
**Pull request by hmadhawa**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,23 @@ Find more plugins [here](https://github.com/prettier/prettier#editor-integration
...
+* Please go through our issue list first(open as well as closed) and make sure the issues you are reporting do not replicate the issues already reported. If you have issues on multiple pages, report them separately. Do not combine them into a single issue.
+* All the PR’s need to be sent to the develop branch.
+
+
+### Avoid doing the following mistakes
+
+* Fix a new issue and submit a PR without reporting and getting it approved at first.
+* Fix an issue assigned to somebody else and submit a PR before the assignee does.
+* Report issues which are previously reported by others. (Please check the closed issues too before you report an issue).
+* Suggest completely new developments in the issue list. (Please use the mailing list/slack channel for this kind of suggestions. Use issue list to suggest bugs/features in the already implemented sections.)
```
Comment (agentmilindu) [inline README.md]: As long as the suggested feature is something related to Bassa, I don't see any harm in having issues for completely new developments.

File: README.md [hunk 2]
```diff
@@ -93,7 +93,23 @@ Find more plugins [here](https://github.com/prettier/prettier#editor-integration
...
+
 	$ cd ui/
 	$ npm test OR $ yarn test 
-```
+
+## How to Contribute
+
+### Contribution Guidelines
+
+* Pick an open issue from the issue list, claim it in the comments,after approval fix the issue and send us a pull request.
```
Comment (agentmilindu) [inline README.md]: Spaces after commas makes is nicer 

`comments,after ` => `comments, after `

Comment (agentmilindu) [inline README.md]: Spaces after commas makes is nicer 

`comments,after ` => `comments, after `

**Comment by agentmilindu** [comment]
@malithsen @rehrumesh Can we merge this PR?

**Comment by kmehant** [comment]
@hmadhawa Thanks for the contribution!
But we have already got contribution guidelines in `develop` branch. [Bassa Contribution Guide](https://github.com/scorelab/Bassa/blob/develop/CONTRIBUTING.md) 
So closing it for now 😃

**Pull request closed without merging**